testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -79,6 +80,8 @@ public void testTrinoReadingSparkData(StorageFormat storageFormat)
         String trinoTableName = trinoTableName(baseTableName);
         String sparkTableName = sparkTableName(baseTableName);
 
+        onTrino().executeQuery("DROP TABLE IF EXISTS " + trinoTableName);
+
```

Comment:
struct/list/map?

Reply:
https://iceberg.apache.org/spark-writes/

Reply:
those are testes in `testTrinoReadingCompositeSparkData` method